### PR TITLE
feature/FOUR-13702: UI: Process Information - Add Header Collapsible

### PR DESCRIPTION
**Pull request by luNunezProcessmaker**
## Issue & Reproduction Steps
image-20240220-184049

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13702

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a2ccc344f7.engk8s.processmaker.net

**Review by ana-mauricio — CHANGES_REQUESTED** [review]
The button Start this Process does not have the white line separation 
image

There are not the same separation between button on the header 
image
 
As show in the figma
@luNunezProcessmaker

File: resources/js/processes-catalogue/components/ProcessCollapseInfo.vue [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+      assetName: "",
+      processLaunchpadActions: [],
+      optionsData: {},
+      infoCollapsed: true,
+      largeDescription: false,
+      readActivated: false,
+      showEllipsis: false,
+      labelTooltip: "",
+      showBookmarkIcon: false,
+      auxBookmarkId: this.process.bookmark_id,
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessCollapseInfo.vue:189]: validate if `bookmark_id` exists

File: resources/js/processes-catalogue/components/ProcessCollapseInfo.vue [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+                @click="checkBookmark(process)"
+              />
+              <i
+                v-else
+                :ref="`bookmark-${process.id}-unmarked`"
+                v-b-tooltip.hover.bottom
+                :title="$t(labelTooltip)"
+                class="fas fa-bookmark unmarked"
+                @click="checkBookmark(process)"
+              />
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessCollapseInfo.vue:58]: the only thing that changes is the class, so 
```javascript
class="fas fa-bookmark"
:class="{ marked: showBookmarkIcon, unmarked: !showBookmarkIcon  }"
```

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a2ccc344f7.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-a2ccc344f7.engk8s.processmaker.net

**Pull request merged**